example/05-project-dev.yaml

```diff
@@ -14,6 +14,6 @@ spec:
   - apiGroup: rbac.authorization.k8s.io
     kind: User
     name: alice.doe@example.com
-# description: "This is my first project"
-# purpose: "Experimenting with Gardener"
-# namespace: "garden-dev" # Optional, will be initialized if unset.
+  # description: This is my first project
+  # purpose: Experimenting with Gardener
+  namespace: garden-dev # Optional, will be initialized if unset.
```

Comment:
Can you add a note here about a random name will be picked if not provided, in my case it was `garden-dev-sws7p`